LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016
+Copyright (c) 2018
```

Comment:
I know this came automatically from ember-cli-update, but I think this should be left untouched.